ddtrace/contrib/django/patch.py

```diff
@@ -139,7 +145,17 @@ def traced_cache(django, pin, func, instance, args, kwargs):
             keys = utils.quantize_key_values(args[0])
             span.set_tag_str("django.cache.key", keys)
 
-        return func(*args, **kwargs)
+        result = func(*args, **kwargs)
+        if command_name == "get_many":
+            span.set_metric(db.ROWCOUNT, len(result) if result and isinstance(result, Iterable) else 0)
+        elif command_name == "get":
+            # if valid result and check for special case for Django~3.0 that returns an empty Sentinel object as
+            # missing key
+            if result and not (hasattr(instance, "_missing_key") and result == instance._missing_key):
```

Comment:
this might be easier read

```suggestion
            if result and result != getattr(instance, "_missing_key", None):
```